appwrite/website
# Add Flutter vs React Native blog post and assets

## What does this PR do?

- Adds a new blog post: Flutter vs React Native

## Test Plan

- Run the site and navigate to `/blog/post/flutter-vs-react-native`
- Confirm that the blog and assets are displayed correctly
